index.d.ts

```diff
@@ -0,0 +1,25 @@
+import { Plugin } from 'webpack';
+
+interface WebpackChunkHashPluginOptions {
+  /**
+   * A callback to add more content to the resulting hash
+   */
+  additionalHashContent?: (_: any) => string;
```

Comment:
Yes, I'm refactoring the build process of our main app and decided to use Typescript in the Webpack configuration to keep the same syntax that the rest of the front-end assests (es2015 exports instead of commonjs) and improve editor's autocomplete to peers that aren't used to webpack.